Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt

```diff
@@ -0,0 +1,147 @@
+                      Analytical Graphics, Inc.
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+           http://www.agi.com/licenses/corporate-cla-agi-v1.0.txt
```

Comment:
I haven't tried this, but maybe worth a shot:  https://github.com/blog/985-git-io-github-url-shortener